recipes/cityhash/meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "python-cityhash" %}
+{% set version = "0.2.3.post9" %}
```

Comment:
Is there a fully released version that can be built?

Reply:
Technically `.post` is fully released.